ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -74,6 +81,32 @@ tests:
     test:
     - ref: windows-e2e-operator-test-with-custom-vxlan-port
     workflow: ipi-vsphere-ovn-hybrid-custom-vxlan-port
+- as: aws-e2e-upgrade
+  steps:
+    cluster_profile: aws
+    env:
+      OO_CHANNEL: preview
```

Comment:
shouldnt this be the stable channel?
https://github.com/openshift/windows-machine-config-operator/blob/master/bundle.Dockerfile#L33

Reply:
That shouldn't matter given a new `CatalogSource` is being created.

Reply:
`operators.operatorframework.io.bundle.channel.default.v1=stable`  label is only for the production build